pyproject.toml

```diff
@@ -71,12 +71,6 @@ apm = [
 em = [
     "pynxtools-em",
 ]
-mpes = [
```

Comment:
these should not be removed, otherwise the plugins will not be automatically installed (e.g. in NOMAD)

Reply:
It was not intended to remove any of the plugin. Unfortunately it occurred in behind the scene probably while rebasing. I added it back.